pkg/broker/receiver.go

```diff
@@ -83,6 +83,10 @@ func (r *Receiver) sendEvent(channel provisioners.ChannelReference, message *pro
 }
 
 func (r *Receiver) getTrigger(ctx context.Context, ref provisioners.ChannelReference) (*eventingv1alpha1.Trigger, error) {
+	// Sadly this doesn't work well because we do not yet have
```

Comment:
FYI @grantr, yet another reason we should update our controller-runtime dependency.